runtime/resources/databases/qdrant/config.yaml

```diff
@@ -0,0 +1,353 @@
+    # `full_scan_threshold_kb`, the query planner will use full-scan search instead of HNSW index
+    # traversal for better performance.
+    # Note: 1Kb = 1 vector of size 256
+    full_scan_threshold_kb: 10000
+
+    # Number of parallel threads used for background index building.
+    # If 0 - automatically select.
+    # Best to keep between 8 and 16 to prevent likelihood of building broken/inefficient HNSW graphs.
+    # On small CPUs, less threads are used.
+    max_indexing_threads: 0
+
+    # Store HNSW index on disk. If set to false, index will be stored in RAM. Default: false
+    on_disk: false
+
+    # Custom M param for hnsw graph built for payload index. If not set, default M will be used.
+    payload_m: null
+
+  # Default shard transfer method to use if none is defined.
+  # If null - don't have a shard transfer preference, choose automatically.
+  # If stream_records, snapshot or wal_delta - prefer this specific method.
+  # More info: https://qdrant.tech/documentation/guides/distributed_deployment/#shard-transfer-method
+  shard_transfer_method: null
+
+  # Default parameters for collections
+  collection:
+    # Number of replicas of each shard that network tries to maintain
+    replication_factor: 1
+
+    # How many replicas should apply the operation for us to consider it successful
+    write_consistency_factor: 1
+
+    # Default parameters for vectors.
+    vectors:
+      # Whether vectors should be stored in memory or on disk.
+      on_disk: null
+
+    # shard_number_per_node: 1
+
+    # Default quantization configuration.
+    # More info: https://qdrant.tech/documentation/guides/quantization
+    quantization: null
+
+      # Default strict mode parameters for newly created collections.
+      #strict_mode:
+      # Whether strict mode is enabled for a collection or not.
+      #enabled: false
+
+      # Max allowed `limit` parameter for all APIs that don't have their own max limit.
+      #max_query_limit: null
+
+      # Max allowed `timeout` parameter.
+      #max_timeout: null
+
+      # Allow usage of unindexed fields in retrieval based (eg. search) filters.
+      #unindexed_filtering_retrieve: null
+
+      # Allow usage of unindexed fields in filtered updates (eg. delete by payload).
+      #unindexed_filtering_update: null
+
+      # Max HNSW value allowed in search parameters.
+      #search_max_hnsw_ef: null
+
+      # Whether exact search is allowed or not.
+      #search_allow_exact: null
+
+      # Max oversampling value allowed in search.
+    #search_max_oversampling: null
+
+  # Maximum number of collections allowed to be created
+  # If null - no limit.
+  max_collections: null
+
+service:
+  # Maximum size of POST data in a single request in megabytes
+  max_request_size_mb: 32
+
+  # Number of parallel workers used for serving the api. If 0 - equal to the number of available cores.
+  # If missing - Same as storage.max_search_threads
+  max_workers: 0
```

Comment:
To operate the API, one thread should suffice in our case, right? After all, we’re only connecting to one client, possibly up to two or three threads at most. I think we can configure a `1` or `2` instead of `0`. What do you think?